tests/mock/clitools.rs

```diff
@@ -637,17 +637,16 @@ where
     let mut vars: HashMap<String, String> = HashMap::default();
     self::env(config, &mut vars);
     vars.extend(env.iter().map(|(k, v)| (k.to_string(), v.to_string())));
-    let mut arg_strings: Vec<Box<str>> = Vec::new();
-    arg_strings.push(name.to_owned().into_boxed_str());
-    for arg in args {
-        arg_strings.push(
+    let arg_strings = Some(name.to_owned().into_boxed_str())
```

Comment:
This is definitely not cleaner: iterating over a Some() to avoid a clippy lint... in test code.